src/lavinmq/amqp/consumer.cr

```diff
@@ -73,8 +73,12 @@ module LavinMQ
           {% end %}
           queue.consume_get(self) do |env|
             deliver(env.message, env.segment_position, env.redelivered)
+            delivered_bytes &+= env.message.bytesize
+          end
+          if delivered_bytes > Config.instance.yield_each_delivered_bytes
```

Comment:
does not have a performance impact to do the double variable dereference in a tight loop?

Reply:
Doesn't seem to make any noticeable impact when benchmarking at least!